sensitivity analysis/sensitivity_analysis.R

```diff
@@ -54,18 +56,26 @@ runlist_final <- c(runlist, params_fixed)
 
 # Run the model
 output <- list()
-for(r in 1:length(runs)){
-  params <- lapply(runlist_final[-1], `[`, r)
-
+for(r in runs){
+  # To access parameter values:
+  # First, we want to drop the run number, so we pass in runlist_final[-1] to
+  # remove the first element of the list.
+  # Then, we need to access the list of parameter values for each run.
+  # We can extract this from the list by looping through each run number and
+  # pulling out that column from each list element. i.e., the first column
+  # corresponds to run 1, second column to run 2, etc.
+  params <- lapply(runlist_final[-1], `[`, r) # could also use mapply for this
```

Comment:
I'm still a tad confused by the ` '['`notation, is this how you index to the parameter values?